src/applications/disability-benefits/all-claims/content/form0781/supportingEvidencePage.jsx

```diff
@@ -0,0 +1,196 @@
+      </li>
+      <li>
+        If you plan to submit these documents later we’ll mail you a letter with
+        instructions once we’ve received your claim.
+      </li>
+    </ul>
+  </>
+);
+
+export const supportingEvidenceBuddyStatement = (
+  <va-additional-info
+    class="vads-u-margin-y--3"
+    trigger="Learn more about buddy statements"
+  >
+    <div>
+      <p className="vads-u-margin-top--0">
+        A buddy statement is a written statement from someone who knows about
+        your condition or related events.
+      </p>
+      <p>
+        Someone can provide a buddy statement to support your claim in either of
+        these 2 ways:
+      </p>
+      <ul>
+        <li>
+          <va-link
+            external
+            href="https://www.va.gov/supporting-forms-for-claims/lay-witness-statement-form-21-10210/introduction"
+            text="Complete VA Form 21-10210 online"
+          />
+          , <strong>or</strong>
+        </li>
+        <li>
+          Give you a PDF version of their completed VA Form 21-10210 to upload
+          on the next screen
+          <va-link
+            external
+            href="https://www.va.gov/supporting-forms-for-claims/lay-witness-statement-form-21-10210/introduction"
+            text="Get VA Form 21-10210 to download"
+          />
+        </li>
+      </ul>
+    </div>
+  </va-additional-info>
+);
+
+export const supportingEvidenceAdditionalInformation = (
+  <va-additional-info
+    class="vads-u-margin-y--3"
+    trigger="Why we’re asking this"
+  >
+    <div>
+      <p className="vads-u-margin-top--0">
+        We understand that traumatic events from your military service may not
+        have been reported or documented. In these situations, the supporting
+        documents you provide will help us understand your situation and
+        identify evidence to support your claim.
+      </p>
+    </div>
+  </va-additional-info>
+);
+
+export const supportingEvidenceNoneLabel =
+  'I don’t have any supporting documents to submit.';
+
+export const supportingEvidenceValidationError = (
+  <va-alert status="error" uswds>
+    <p className="vads-u-font-size--base">
+      You selected one or more types of reports for this event, but you also
+      selected ‘I don’t have any supporting documents to submit.’
+    </p>
+    <p>Revise your selection so they don’t conflict to continue.</p>
+  </va-alert>
+);
+
+/**
+ * Returns true if 'none' selected, false otherwise
+ * @param {object} formData
+ * @returns {boolean}
+ */
+function hasSelectedNoneCheckbox(formData) {
+  return Object.values(formData.supportingEvidenceNoneCheckbox || {}).some(
+    selected => selected === true,
+  );
+}
+
+/**
+ * Returns an object with evidence properties and boolean value if selections present within each section
+ * @param {object} formData
+ * @returns {object}
+ */
+function selectedDocumentsSections(formData) {
```

Comment:
I believe `supportingEvidenceUnlisted` will also need to be added to this and the `hasSelectedSupportingEvidence()` function.

Reply:
Missed adding to `validateSupportingEvidenceSelections()` function, but I believe error messaging is being updated in https://github.com/department-of-veterans-affairs/va.gov-team/issues/104597, regardless.